alphagov/paas-product-pages
# Migration guidance

Initial version of some migration guidance for tenants migrating to another platform

## how to review
- clone the branch run locally using the instructions in the README
- check the copy for typos and syntax
- ensure all links work correctly 


## who can review

- anyone but @pauldougan 
- @wryobservations
